### PR TITLE
ciao(bye): bisou

### PR DESCRIPTION
**Pull request by cgagnaire**
## Description

See ya!

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [x] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by adr-mo** [comment]
😢

**Review by adr-mo — DISMISSED** [review]

**Comment by Nohzoh** [comment]
Bonne continuation Colin.

**Review by Nohzoh — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

File: www/include/Administration/about/about.php [hunk 1]
```diff
@@ -80,7 +80,6 @@
                     <p>Adrien Gelibert</p>
                     <p>Adrien Morais</p>
                     <p>Bruno D'auria</p>
-                    <p>Colin Gagnaire</p>
```
Comment (sc979) [inline www/include/Administration/about/about.php:83]: Why delete your name ? Move it in the contributors below, you earned it ;)

**Review by sc979 — DISMISSED** [review]

**Comment by jdelpierre** [comment]
Bonne continuation !

**Review by Thebarda — DISMISSED** [review]
😢 😢 😢 😢 Bonne continuation M'sieur !

File: www/include/Administration/about/about.php (hunk 1 above)
Comment (cgagnaire) [reply www/include/Administration/about/about.php:83]: Yes I though about it but I didn't understand how they are organised. I'll put myself at the end of the list.

**Review by adr-mo — APPROVED** [review]

**Review by Nohzoh — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Review by callapa — APPROVED** [review]
Bonne continuation Colin (tout schuss !)

**Pull request merged**